playbooks/common/openshift-cluster/initialize_facts.yml

```diff
@@ -6,12 +6,140 @@
 
 - name: Initialize host facts
   hosts: oo_all_hosts
-  roles:
-  - openshift_facts
   tasks:
-  - openshift_facts:
+  - name: load openshift_facts module
+    include_role:
+      name: openshift_facts
+
+  # TODO: Should this role be refactored into health_checks??
+  - name: Run openshift_sanitize_inventory to set variables
+    include_role:
```

Comment:
I will be always opposed to that as long as the original `roles` property of a play can provide the same functionality. Though, it is my own opinion and not a blocker for the PR :).